VKAPI: методы для подарков, заметок, статусов, обсуждений и немного для фоток и групп

lalka2016 opened the pull request:
Данный пулл-реквест добавляет api-методы для:

- Подарков (get, send). Чтобы отобразить список подарков, добавлены 2 метода, которых нет в вк апи: getCategories и getGiftsInCategory
- Заметок (add, createComment, delete, deleteComment, edit, editComment, get, getById, getComments)
- Статусов (get, set)
- Пару методов для групп (create, edit, getMembers, getSettings, isMember)
- Совместимое с вк-апи прикрепление фотки к посту (работает только с одним прикреплением)

Commit:
API methods for gifts, notes, statuses

Commit:
Some fixes

Строки локализации у gifts.send теперь не костыльные и можно прикрепить до 10 аттачей к посту

veselcraft reviewed (approved):
вроде норм глянем че другие чилипиздрики скажут

Lumaeris reviewed (approved):
asdfghbjnkml,swedrft

*игнорьте этот пост, я отправил по приколу*

celestora reviewed (changes requested):
Заметки не были позырканы, есть некоторые замечания по кодстайлу и пагинации

Commit:
Small imp

Пофиксил пагинацию у заметков и подарок
Перенёс структуру заметок
Добавил аттачи к комментариям
Добавил проверку на удалённость аттача
Ну и пофиксил сортировку заметок

Commit:
Merge branch 'master' into newapis

Commit:
VKAPI: Some methods for topics and photos

Добавлены методы для обсуждений (addTopic, closeTopic(), createComment(), deleteComment(), deleteTopic(), editTopic(), fixTopic(), getComments(), getTopics(), openTopic(), unfixTopic())
и для фотографий (createAlbum(), editAlbum(), getAlbums(), getAlbumsCount(), getById(), get(), deleteAlbum(), edit(), delete(), deleteComment(), createComment(), getAll(), getComments())

Commit:
Merge branch 'newapis' of https://github.com/lalka2016/openvkuh into newapis

Commit:
fixsex

celestora reviewed (commented):
мне лень проверять но чел вроде ровный, доверять можно 👍🏻
